src/common/helpers/auth/get-entra-user-roles.js

```diff
@@ -0,0 +1,23 @@
+import Boom from '@hapi/boom'
+import { getConfig } from '../../../config.js'
+
+export async function getEntraUserRoles(tokenPayload, request) {
+  const userEmail = tokenPayload.email || tokenPayload.preferred_username
+
+  const stringifiedUserRoles = getConfig().get('userRoles')
+
+  let userRoles = {}
```

Comment:
I think this needs to be an array `[]` rather than a `{}` based on its usage in the `validate` function of the jwt plugin config

```js
        return {
          isValid: true,
          credentials: {
            id: contactId,
            email,
            issuer,
            scope: await getEntraUserRoles(tokenPayload)
          }
        }
```

Reply:
`thisUserRoles` is an array. `userRoles` is just the JSON parsed from the stringified env var.